failed to initialise OpenGL

`
package main

import (
	"fyne.io/fyne/app"
	"fyne.io/fyne/widget"
)

func main() {
	app := app.New()
	w := app.NewWindow("Hello")
	w.SetContent(widget.NewVBox(
		widget.NewLabel("Hello Fyne!"),
		widget.NewButton("Quit", func() {
			app.Quit()
		}),
	))
	w.ShowAndRun()
}
`
Output;
PS C:\GoCode\fyne> go run .\fyne_sample.go
2019/09/04 19:16:37 Fyne error:  failed to initialise OpenGL
2019/09/04 19:16:37   Cause: glGetBufferParameteri64v
2019/09/04 19:16:37   At: C:/Users/eduar/go/src/fyne.io/fyne/internal/driver/gl/window.go:1007
panic: runtime error: invalid memory address or nil pointer dereference
[signal 0xc0000005 code=0x0 addr=0x54 pc=0x5b394d]

--- a/internal/driver/glfw/glfw_core.go
+++ b/internal/driver/glfw/glfw_core.go
@@ -10,7 +10,7 @@ import "github.com/go-gl/glfw/v3.3/glfw"
 
 func initWindowHints() {
 	glfw.WindowHint(glfw.ContextVersionMajor, 2)
-	glfw.WindowHint(glfw.ContextVersionMinor, 0)
+	glfw.WindowHint(glfw.ContextVersionMinor, 1)
 
 	glfw.WindowHint(glfw.CocoaGraphicsSwitching, glfw.True)
 }

--- a/internal/driver/glfw/glfw_es.go
+++ b/internal/driver/glfw/glfw_es.go
@@ -12,5 +12,5 @@ import "github.com/go-gl/glfw/v3.3/glfw"
 func initWindowHints() {
 	glfw.WindowHint(glfw.ClientAPI, glfw.OpenGLESAPI)
 	glfw.WindowHint(glfw.ContextVersionMajor, 2)
-	glfw.WindowHint(glfw.ContextVersionMinor, 0)
+	glfw.WindowHint(glfw.ContextVersionMinor, 1)
 }

--- a/internal/painter/gl/gl_core.go
+++ b/internal/painter/gl/gl_core.go
@@ -6,7 +6,7 @@ package gl
 import (
 	"strings"
 
-	"github.com/go-gl/gl/v3.2-core/gl"
+	"github.com/go-gl/gl/v2.1/gl"
 
 	"fyne.io/fyne/v2"
 )

